runtime/queries/nu/injections.scm

```diff
@@ -0,0 +1 @@
+(comment) @comment
```

Comment:
nvim-treesitter does injections differently as well. For helix this would be:

```scm
((comment) @injection.content
 (#set! injection.language "comment"))
```